### PR TITLE
Implement some common keybinds:

### PR DESCRIPTION
**Pull request by Carpetsmoker**
- ^L to clear screen
- ^K to delete all text after the cursor
- ^E to go to the end of the line
- ^A to go to the start of the line
- ^C to delete the entire line (without executing it)

### CODE REVIEW

File: lib/better_errors/templates/main.erb [hunk 1]
```diff
@@ -915,6 +939,21 @@
...
+        } else if (ev.keyCode == 65 && ev.ctrlKey) { // ^A
+            ev.preventDefault();
+            this.beginningOfLine();
+        } else if (ev.keyCode == 69 && ev.ctrlKey) { // ^E
+            ev.preventDefault();
+            this.endOfLine();
+        } else if (ev.keyCode == 75 && ev.ctrlKey) { // ^K
+            ev.preventDefault();
+            this.killLine()
+        } else if (ev.keyCode == 67 && ev.ctrlKey) { // ^C
```
Comment (charliesome) [inline lib/better_errors/templates/main.erb]: Won't this interfere with Linux/OS X users trying to copy sections of text from the input box?

**Comment by charliesome** [comment]
I'm not really sure if Better Errors should implement these keys itself. Adding special keybindings also introduces the possibility that Better Errors could conflict with custom keybindings the user has, which makes me uncomfortable.

**Comment by Carpetsmoker** [comment]
You're right that this "overrides" browser settings, I think this is a good thing in this case, since I'm in '*console mode*', and not '*browser mode*' when I'm editing stuff. I mis these keybinds a lot...
However, I can imagine that  other people might have different opinions.

I originally looked at making this more configurable, but wasn't sure on how to do that... 

Maybe something in a config like:

	BetterErrors.editing_options = {
		some_other_option_done_later: true,

		keybinds: [
			{
				keyCode: 76,
				ctrlKey: true,
				action: 'this.clearLine()'
			},
			{
				keyCode: 75,
				ctrlKey: true,
				action: 'this.killLine()',
			}
		]
	}

But suggestions welcome, of course :-)

**Comment by Carpetsmoker** [comment]
You're right that this "overrides" browser settings, I think this is a good
thing in this case, since I'm in '*console mode*', and not '*browser mode*' when
I'm editing stuff, and miss these sort of keybinds a lot (these are only the
basics, and would like to implement more).

I can imagine that other people might have different opinions, and those are
reasonable as well.

I originally looked at making this more configurable, but wasn't sure on how to do that... 

Maybe something in a config like:

	BetterErrors.options = {
		other_option: true,
	
		bindkeys: [
				{
						keyCode: 76,
						ctrlKey: true,
						action: 'this.clearLine()'
				},
				{
						keyCode: 75,
						ctrlKey: true,
						action: 'this.killLine()',
				}
		]
	}


Suggestions welcome, of course...

**Pull request closed without merging**